bip-update-process.md

```diff
@@ -0,0 +1,705 @@
+```
+BIP: ?
+Title: Updated BIP Process
+Author: Murch <murch@murch.one>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-Updated-BIP-Process
+Status: Draft
+Type: Process
+Created: 2024-05-13
+License: BSD-2-Clause
+Post-History: https://github.com/murchandamus/bips/pull/2
+Requires: 123
+Replaces: 2
+```
+
+## Abstract
+
+This _Bitcoin Improvement Proposal (BIP)_ provides information about the preparation of BIPs and policies relating to
+the publication of BIPs. It replaces [BIP 2](bip-0002.mediawiki) with a streamlined process, and may be amended to
+address the evolving needs of the BIP process.
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the bitcoin[^capitalization]
+community. The BIP process as defined by BIP 2 thus far seems to have been fashioned to facilitate the design and
+activation of protocol changes. In the past years, BIPs more often describe interoperable features beyond the base
+protocol. The community has had multiple debates about the role of BIP Editors, and some aspects of the process
+specified by BIP 2 that did not seem to achieve the intended goals. This proposal sunsets aspects of the BIP 2 process
+that did not achieve broad adoption, reduces the judgment calls assigned to the BIP Editor role, and delineates the
+BIP Types more expediently.
+
+## Fundamentals
+
+### What is a BIP?
+
+BIPs cover the range of interests of the bitcoin community. The main topic is technology that supports the bitcoin
+currency. Most BIPs provide a concise, self-contained, technical description of one new concept, feature, or standard.
+Some BIPs describe processes, implementation guidelines, best practices, incident reports (e.g.
+[BIP 50](bip-0050.mediawiki)), or other information relevant to the bitcoin community. However, any topics related to
+the bitcoin protocol, bitcoin’s peer-to-peer network, and bitcoin client software may be acceptable.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into implementations. BIPs may be submitted by anyone.
+
+### BIP Ownership
+
+Each BIP is primarily owned by its authors and represents the authors’ opinion or recommendation. The authors are
+expected to foster discussion, address feedback and dissenting opinions, and, if applicable, advance the adoption of
+their proposal within the bitcoin community.
+
+#### Authors and Shepherds
+
+Authors may want additional support with the BIP process after writing an initial draft. In that case, they may assign
+one or more Shepherds to their BIP. Shepherds are stand-in owners of a BIP who were not involved in writing the
```

Comment:
The distinction can make a difference when it comes to copyright. I agree that a single role is simpler in terms of the process, but I believe this is what is implemented here. We have effectively a single Owner *role* (and the Owners are the union of Authors and Shepherds), but additionally an author *field* (which is pure metadata and doesn't have implications for the process). 

Now, I agree that this is a bit difficult to explain... 

Perhaps there should just be two required fields "Owners" and "Authors". This sounds like overkill because it leads to duplication. But if you think about it, I believe it's simpler than the current draft:  it avoids the term Shepherd entirely, and in the text, you can easily pick the appropriate term on a case-by-case basis.

Moreover, there may be an author who is not an owner (anymore). Not sure if we'll ever need this, but there's no good reason to exclude it upfront.

Reply:
I added the additional role per the request of several reviewers. I don’t have strong feelings about this part: I would be happy with just "Owners" or "Authors", I can also live with two roles. It seems to me that people might still be discovering their positions on this aspect, so if anyone has strong feelings, please feel free to discuss further, but I’m gonna give this discussion some time to develop before making additional changes.